ci/refresh-testnet.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+#
+# Refreshes the Solana software running on the Testnet full nodes
+#
+# This script must be run by a user/machine that has successfully authenticated
+# with GCP and has sufficient permission.
+#
+
+if [[ -z $SOLANA_METRICS_CONFIG ]]; then
+  echo Error: SOLANA_METRICS_CONFIG environment variable is unset
+  exit 1
+fi
+
+# Default to --edge channel.  To select the beta channel:
+#   export SOLANA_METRICS_CONFIG=--beta
```

Comment:
When we're ready to flip to a tagged release, `SOLANA_METRICS_CONFIG=--beta` gets defined in the buildkite pipeline configuration